doc/userguide/rules/multi-buffer-matching.rst

```diff
@@ -0,0 +1,64 @@
+Multiple Buffer Matching
+========================
+
+Suricata 7 and newer now supports matching contents in multiple buffers within the
+same transaction.
+
+Using a single DNS transaction that has two queries in it:
+
+query 1: [example.net]
+query 2: [something.com]
+
+Example rule:
+
+.. container:: example-rule
+
+    `alert dns $HOME_NET any -> $EXTERNAL_NET any (msg:"DNS Multiple Question Example Rule"; dns.query; content:"example"; dns.query; content:".com"; sid:1;)`
```

Comment:
Thanks for this @jmtaylor90 

Would you also add an example with 
`dns.query; content:"example"; content:".com";` ?
And say that these will match on the same dns query buffer (or a better wording)
(maybe this is better illustrated with http2.header)

Reply:
Yes, I will add and look at the explanation again today. I went back and forth a bit on which example to use for a signature, I started with using a variation of the signature from your suricata-verify test for the http2 multi buffer but I wasn't sure if the concept remained clear throughout the explanation of the behavior changes.
Thanks for the review/comments.